x/concentrated-liquidity/incentives.go

```diff
@@ -1032,3 +1045,50 @@ func emitAccumulatorUpdateTelemetry(ctx sdk.Context, truncatedPlaceholder, emitt
 		ctx.Logger().Info(emittedPlaceholder, "pool_id", poolID, "total_liq", liquidityInAccum, "per_unit_liq", rewardsPerUnitOfLiquidity, "total_amt", rewardsTotal, extraKeyVals)
 	}
 }
+
+// getIncentiveScalingFactorForPool returns the scaling factor for the given pool.
+// It returns perUnitLiqScalingFactor if the pool is migrated or if the pool ID is greater than the migration threshold.
+// It returns oneDecScalingFactor otherwise.
+func (k Keeper) getIncentiveScalingFactorForPool(ctx sdk.Context, poolID uint64) (osmomath.Dec, error) {
+	migrationThreshold, err := k.GetIncentivePoolIDMigrationThreshold(ctx)
+	if err != nil {
+		return osmomath.Dec{}, err
+	}
+
+	// If the given pool ID is greater than the migration threshold, we return the perUnitLiqScalingFactor.
+	if poolID > migrationThreshold {
```

Comment:
sanity checking we got > vs >= right?

Reply:
It is set to `lastPoolID`
https://github.com/osmosis-labs/osmosis/blob/68020b18694660308e1fd6e3ad9c505ca031498a/app/upgrades/v23/upgrades.go#L36-L40

Reply:
Looks right to me too